iroh-bytes/src/protocol.rs

```diff
@@ -22,11 +21,14 @@ pub(crate) const MAX_MESSAGE_SIZE: usize = 1024 * 1024 * 100;
 /// Protocol version
 pub const VERSION: u64 = 2;
 
-#[derive(Deserialize, Serialize, Debug, PartialEq, Eq, Clone, MaxSize)]
+#[derive(Deserialize, Serialize, Debug, PartialEq, Eq, Clone)]
 pub(crate) struct Handshake {
     pub version: u64,
 }
 
+/// Serialized length of [Handshake].
+pub(crate) const HANDSHAKE_SIZE: usize = 1;
```

Comment:
That's a nice handshake size. But what will we do after version 255? Maybe we should do a multihandshake crate 🤣 ...

Reply:
Sorry, I actually ended up increasing the size, this comment made me realize that postcard under the hood still uses varints.. Now we have a fully hard coded handshake, with a lot of protocol versions left 😅